src/IR.h

```diff
@@ -814,6 +814,24 @@ struct Prefetch : public StmtNode<Prefetch> {
     static const IRNodeType _node_type = IRNodeType::Prefetch;
 };
 
+/** Lock all the Store nodes in the body statement.
```

Comment:
Nit: our multiline comments include a `*` at the start of each line

Reply:
fixed